src/store/conversationsStore.js

```diff
@@ -216,11 +216,7 @@ const mutations = {
 	},
 
 	overwriteHasCallByChat(state, { token, hasCall }) {
-		if (hasCall) {
-			Vue.set(state.conversations[token], 'hasCallOverwrittenByChat', hasCall)
-		} else {
-			Vue.delete(state.conversations[token], 'hasCallOverwrittenByChat')
-		}
+		Vue.set(state.conversations[token], 'hasCallOverwrittenByChat', hasCall)
 	},
```

Comment:
So why are you keeping it then ?

Reply:
good question. I thought to mark the conversation as modified, until we fetch a proper one from server